src/daemon/keymap_patch.c

```diff
@@ -16,6 +16,14 @@ keypatch k63patch[] = {
 };
 #define K63PATCH_LEN sizeof(k63patch)/sizeof(*k63patch)
 
+keypatch m55patch[] = {
+    { 210, "mouse6", -1, KEY_CORSAIR },
+    { 211, "mouse7", -1, KEY_CORSAIR },
+    { 213, "dpiup",  -1, KEY_CORSAIR },
+    { 237, "dpi", LED_MOUSE, KEY_NONE },
```

Comment:
Please at least zero out the existing dpi entry as well, because as it is, there are two "dpi" leds with this patch applied. This might also be the reason the DPI led wasn't updating correctly.

Reply:
Shortened LUT to 8 entries